source/operations/install-deploy-manage/migrate-fs-gateway.rst

```diff
@@ -0,0 +1,203 @@
+
+   Use :mc-cmd:`mc admin tier ls` with the ``--json`` flag to retrieve a list of the tiers that exist on the standalone deployment.
+
+   .. code-block:: shell
+      :class: copyable
+
+      mc admin tier ls ALIAS --json
+
+   - Replace ``ALIAS`` with the alias for the existing standalone deployment.
+   
+   Use the list to recreate the tiers on the new deployment.
+
+#. Duplicate **policies** from existing standalone deployment to new deployment
+
+   Use :mc-cmd:`mc admin policy list` with the ``--json`` flag  to retrieve a list of policies that exist on the standalone deployment.
+   
+   .. code-block:: shell
+      :class: copyable
+
+      mc admin policy list ALIAS --json
+
+   - Replace ``ALIAS`` with the alias for the existing standalone deployment.
+
+   Use the list to recreate the policies on the new deployment.
+
+#. Duplicate **groups** from existing standalone deployment to new deployment
+
+   Use :mc-cmd:`mc admin group list` with the ``--json`` flag to retrieve a list of groups that exist on the standalone deployment.
+
+   .. code-block:: shell
+      :class: copyable
+
+      mc admin group list ALIAS --json
+
+   - Replace ``ALIAS`` with the alias for the existing standalone deployment.
+
+   Use the list to recreate the groups on the new deployment.
+
+#. Duplicate **users** from existing standalone deployment to new deployment
+
+   Use :mc-cmd:`mc admin user list` with the ``--json`` flag to retrieve a list of users with access key, policy name, and status.
+
+   .. code-block:: shell
+      :class: copyable
+
+      mc admin user list ALIAS --json
+
+   - Replace ``ALIAS`` with the alias for the existing standalone deployment.
+
+   Use the list to recreate the users on the new deployment.
+
+   Note: You will need to define each user's ``SECRETKEY`` on the new deployment.
+   Make note of the ``SECRETKEY`` for each user to let them know their new credentials.
+
+#. Duplicate **service accounts** from existing standalone deployment to new deployment
+
+   Use :mc-cmd:`mc admin user svcacct list` with the ``--json`` flag to list existing service accounts on the standalone deployment.
+
+   .. code-block:: shell
+      :class: copyable
+
+      mc admin user svcacct list ALIAS --json
+
+   - Replace ``ALIAS`` with the alias for the existing standalone deployment.
+
+   Use the list to recreate the service accounts on the new deployment.
+
+#. Use :mc:`mc mirror` with the :mc-cmd:`~mc mirror --preserve` and :mc-cmd:`~mc mirror --watch` flags on the standalone deployment to move objects to the new |SNSD| deployment
+
+   .. code-block:: shell
+      :class: copyable
+
+      mc mirror --preserve --watch SOURCE/BUCKET TARGET/BUCKET
+
+   - Replace ``SOURCE/BUCKET`` with the alias and a bucket for the existing standalone deployment.
+   - Replace ``TARGET`` with the alias and corresponding bucket for the new deployment.
+
+#. After the initial mirror process completes, convert standalone deployment to be read only
+
```

Comment:
We could address Eco's point by simply noting "If the legacy deployment does not have any active write operations, you can skip this step and let `mc mirror` complete as per the next step."

Reply:
I simplified the step to just stopping writes from S3 or POSIX clients.